splice_machine/src/main/java/com/splicemachine/derby/stream/output/delete/DeletePipelineWriter.java

```diff
@@ -44,23 +44,27 @@
     public int rowsDeleted = 0;
     protected DeleteOperation deleteOperation;
 
-    public DeletePipelineWriter(TxnView txn,byte[] token,long heapConglom,long tempConglomID, String tableVersion, ExecRow execRowDefinition, OperationContext operationContext) throws StandardException {
-        super(txn, token, heapConglom, tempConglomID, tableVersion, execRowDefinition, operationContext);
+    public DeletePipelineWriter(TxnView txn, byte[] token, long heapConglom, long tempConglomID,
+                                String tableVersion, ExecRow execRowDefinition, OperationContext operationContext,
+                                boolean loadReplaceMode)
+            throws StandardException {
+        super(txn, token, heapConglom, tempConglomID, tableVersion, execRowDefinition, operationContext, loadReplaceMode);
         if (operationContext != null) {
             deleteOperation = (DeleteOperation)operationContext.getOperation();
         }
     }
 
     public void open() throws StandardException {
-        open(deleteOperation != null ? deleteOperation.getTriggerHandler() : null, deleteOperation);
+        open(deleteOperation != null ? deleteOperation.getTriggerHandler() : null, deleteOperation, loadReplaceMode);
     }
 
-    public void open(TriggerHandler triggerHandler, SpliceOperation operation) throws StandardException {
-        super.open(triggerHandler, operation);
+    public void open(TriggerHandler triggerHandler, SpliceOperation operation, boolean loadReplaceMode) throws StandardException {
+        super.open(triggerHandler, operation, loadReplaceMode);
         try{
             WriteConfiguration writeConfiguration = writeCoordinator.defaultWriteConfiguration();
             if(rollforward)
                 writeConfiguration = new RollforwardWriteConfiguration(writeConfiguration);
+            writeConfiguration.setLoadReplaceMode(loadReplaceMode);
```

Comment:
Isn't it possible you are changing here a shared writeConfiguration?

Reply:
you're right, i replaced with newDefaultWriteConfiguration, which returns a new object